docker/node/bin/run-test-e2e

```diff
@@ -7,7 +7,8 @@ if
   [[ "--components-angular" == "${1}" ]] ||
   [[ "--components-react" == "${1}" ]] ||
   [[ "--components-vue" == "${1}" ]] ||
-  [[ "--storefront" == "${1}" ]]
+  [[ "--storefront" == "${1}" ]] ||
```

Comment:
If we would stick to the same order everywhere, then crawler should come after components-vue, right?